addons/point_of_sale/models/pos_printer.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+class PosPrinter(models.Model):
+
+    _name = 'pos.printer'
+    _description = 'Point of Sale Printer'
+
+    name = fields.Char('Printer Name', required=True, default='Printer', help='An internal identification of the printer')
+    printer_type = fields.Selection(string='Printer Type', default='iot',
+        selection=[('iot', ' Use a printer connected to the IoT Box')])
+    proxy_ip = fields.Char('Proxy IP Address', help="The IP Address or hostname of the Printer's hardware proxy")
```

Comment:
Should be required at some point? Isn't it?

Reply:
It seems that putting required=True on a fields.Selection generates an error, could you check this with me later?